snuba/replacers/errors_replacer.py

```diff
@@ -263,6 +262,10 @@ def get_timestamp_condition(msg_value: Optional[str], operator: str) -> str:
         if not msg_value:
             return ""
 
+        if "+" in msg_value:  # e.g. "2023-08-28T03:05:38+00:00"
```

Comment:
I would feel more comfortable if we tried to parse the timestamp with the other format instead and then re-serialize it into what we want. This is a little too hacky for my taste